templates/header/package_navigation.html

```diff
@@ -85,6 +86,15 @@ <h1 id="crate-title">
                                 <span class="title"> Builds</span>
                             </a>
                         </li>
+
+                        {# The features tab #}
+                        <li class="pure-menu-item">
+                            <a href="/crate/{{ crate_path | safe }}/features"
```

Comment:
`| safe` is the default, you don't need to restate it. https://tera.netlify.app/docs/#auto-escaping
```suggestion
                            <a href="/crate/{{ crate_path }}/features"
```

Reply:
`safe` declares the string to have already been escaped and _disables_ the auto-escaping.

Reply:
Hmm, ok - @almusil why did you disable the escaping?

Reply:
I'll be honest, it was copied from the builds link.

Reply:
Ok, it was probably switched in https://github.com/rust-lang/docs.rs/pull/906 then. No need to change it.